waspc/data/Generator/templates/sdk/wasp/client/operations/actions/core.ts

```diff
@@ -32,5 +37,12 @@ export function createAction(relativeActionRoute, entitiesUsed) {
   const action = (args) => internalAction(args, [])
   action.internal = internalAction
 
-  return action
+  return action as ActionFor<BackendAction>
```

Comment:
Needed to add this to avoid type errors.

I don't think a comment is necessary, the file is not migrated fully to TS anyway. However does that one day will know what to do.